scripting-docs/javascript/javascript-in-vs-2017.md

```diff
@@ -200,6 +200,34 @@ You could use the following `tsconfig.json` to make sure the language service on
   }
 }
 ```
+# Troubleshooting The JavaScript language service has been disabled for the following project(s)
+When opening a JavaScript project that has a huge amount of content you might get a message that reads "The JavaScript language service has been disabled for the following project(s)". The most common reason for having so much .js source is due to including libraries with a ton of content.
```

Comment:
"a ton of content" is not very informative. Can you be more specific? (e.g. "more than 20MB or source code", or just "more source code than the internal maximum in place to limit memory consumption").

Reply:
Changed text to be more descriptive.